### PR TITLE
Install instructions for source build

### PR DESCRIPTION
**Pull request by raffaem**
Instructions on how to install the software if you build from source

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,15 @@ Install the Go compiler from [golang.org](https://golang.org/). And make sure yo
...
- ```
+```
+
+To install it:
+
+```shell
+cp ./bin/noisetorch ~/.local/bin
+cp ./assets/noisetorch.desktop ~/.local/share/applications
+cp ./assets/icon/noisetorch.png ~/.local/share/icons/hicolor/256x256/apps
+```
```
Comment (AXDOOMER) [inline README.md:131]: I'd like you to move it just above the `Uninstall` section. I think that would be the best place to put this section. What do you think?

**Review by AXDOOMER — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raffaem) [reply README.md:131]: Doesn't the "Download & Install" section refers to an installation from binary release?

Comment (AXDOOMER) [reply README.md:131]: Oh you're right. Sorry.

**Review by AXDOOMER — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AXDOOMER) [reply README.md:131]: I can't mark this thread as resolved sadly, but I approved your changes.

**Review by Technetium1 — COMMENTED** [review]
LGTM

**Review by ZyanKLee — APPROVED** [review]

**Pull request merged**